go/service/account.go

```diff
@@ -260,3 +260,50 @@ func (h *AccountHandler) TimeTravelReset(ctx context.Context, arg keybase1.TimeT
 
 	return err
 }
+
+/////////////////////////
+type GetContactSettingsResponse struct {
+	libkb.AppStatusEmbed                                /// ?
```

Comment:
is this necessary

Reply:
yes, to grab the status code and error message.